### PR TITLE
test

### PR DESCRIPTION
**Pull request by WaveOfDandelions**

### CODE REVIEW

File: src/convertToObject.js [hunk 1]
```diff
@@ -3,16 +3,22 @@
...
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .trim()
+    .split(';')
+    .map((item) => item.trim().split(':'))
+    .filter((item) => item.length > 1)
+    .reduce(function(prev, item) {
+      prev[item[0].trim()] = item[1].trim();
```
Comment (choeqq) [inline src/convertToObject.js:18]: you can desctructure both key and value from item

**Review by choeqq — CHANGES_REQUESTED** [review]

File: src/convertToObject.js [hunk 2]
```diff
@@ -3,16 +3,22 @@
...
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .trim()
+    .split(';')
+    .map((item) => item.trim().split(':'))
+    .filter((item) => item.length > 1)
+    .reduce(function(prev, item) {
```
[the remark this answers is not in the document]
Comment (WaveOfDandelions) [reply src/convertToObject.js:17]: fixed

File: src/convertToObject.js (hunk 1 above)
Comment (WaveOfDandelions) [reply src/convertToObject.js:18]: > you can desctructure both key and value from item

Can you provide an example?

Comment (JoyCoffeeAddict) [reply src/convertToObject.js:18]:
```js 
const [key, value] = item
```

**Review by asakevych — APPROVED** [review]
Great job!! 🚀 🔥 👏

**Pull request closed without merging**